AltStore/Operations/FetchProvisioningProfilesOperation.swift

```diff
@@ -268,8 +268,17 @@ extension FetchProvisioningProfilesOperation
                             }
                         }
                     }
+                    //App ID name must be ascii. If the name is not ascii, using bundleID instead
+                    let appIDName: String
+                    if !name.allSatisfy({ $0.isASCII }) {
+                        //Contains non ASCII (Such as Chinese/Japanese...), using bundleID
+                        appIDName = bundleIdentifier
```

Comment:
I think it will be better if it can ask users for a new name.

Reply:
Most users probably don't want to enter a new name, or don't know what an app ID is. However, we can easily add this as an unstable feature available in Developer Mode.